docs/index.md

```diff
@@ -12,18 +12,19 @@ SPDX-License-Identifier: MPL-2.0
 :align: right
 ```
 
-`power-grid-model` is a library for steady-state distribution power system analysis distributed for Python and C.
+`power-grid-model` is a library for steady-state distribution power system analysis.
+It is distributed for Python and C.
 The core of the library is written in C++.
 Currently, it supports both symmetric and asymmetric calculations for the following calculation types:
 
 * Power Flow
 * State Estimation
 * Short Circuit
 
-For various conversions to the power-grid-model, refer to the [power-grid-model-io](https://github.com/PowerGridModel/power-grid-model-io) repository.
+For conversions from various data format to the power-grid-model, refer to the [power-grid-model-io](https://github.com/PowerGridModel/power-grid-model-io) repository.
```

Comment:
Shouldn't it be from data format to data format? instead of from data foramt to pgm? Maybe just keep the original one since it's short and clear enough

Reply:
I would argue that the logic here is we convert the data that pgm could use, hence conversion from data format to pgm. But you are right pointing that out. The following would more accurate (but more lengthy):
```suggestion
For conversions from various data format to power-grid-model data, refer to the [power-grid-model-io](https://github.com/PowerGridModel/power-grid-model-io) repository.
```
However, I do not think the original one is `clear enough`.